s9/map.rb

```diff
@@ -0,0 +1,36 @@
+# .map
+num = [1,2,3,4,5]
+
+q = []
+num.each {|n| q << n ** 2}
+p q
+
+sq = num.map{|n| n ** 2}
```

Comment:
What's the difference between `.each` and `.map`?

Reply:
Each method iterates and allows you to operate on each item within the block, but the Map method extracts the returns the expression within the block utilizing an Array.